model/event/DeliveryCreatedEvent.php

```diff
@@ -59,7 +69,7 @@ public function jsonSerialize()
      */
     public function getWebhookEventName()
     {
-        return 'DeliveryCreatedEvent';
+        return 'delivery-published';
```

Comment:
Based on our discussion, you need to revert this modification, right?

Reply:
yes